src/views/SinglePostView.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import axios from 'axios'
+import LoaderSpinner from '../components/LoaderSpinner.vue'
+import PostComment from '../components/PostComment.vue'
+import type { Post, Comment } from '../types/interfaces'
+
+const route = useRoute()
+const post = ref<Post | null>(null)
+const comments = ref<Comment[]>([])
+const isLoading = ref(true)
+
+onMounted(async () => {
+  const postId = route.params.id
+  try {
+    const postResponse = await axios.get(`https://jsonplaceholder.typicode.com/posts/${postId}`)
+    post.value = postResponse.data
+
+    const commentsResponse = await axios.get(
+      `https://jsonplaceholder.typicode.com/posts/${postId}/comments`
+    )
+    comments.value = commentsResponse.data
+  } catch (error) {
+    console.error('Error fetching post or comments:', error)
+  } finally {
+    isLoading.value = false
+  }
+})
+</script>
+
+<template>
+  <div>
+    <LoaderSpinner v-if="isLoading" />
+    <div v-else-if="post">
+      <h1>{{ post.title }}</h1>
+      <p>{{ post.body }}</p>
+
+      <div class="separator"></div>
+
+      <div v-if="comments.length > 0">
+        <PostComment v-for="comment in comments" :key="comment.id" :comment="comment" />
+      </div>
+      <div v-else>
+        <p>Este post aun no tiene comentarios</p>
+      </div>
+    </div>
+    <div v-else>
+      <p>Post no encontrado</p>
+    </div>
+  </div>
+</template>
```

Comment:
Not a big deal but we were expecting a different design here, having a title for the comments por example, differentiate them, showing the Post author, etc. Small little things that would make this page less caotic.

Reply:
We were lacking some functionality here from the assignment:

- En clicar sobre el nom de l'usuari (des d'un post o un comentari), es navegarà a la pàgina de perfil d'aquest usuari.

- També es mostrarà el llistat de comentaris associats al post, amb la possibilitat d'afegir un nou comentari.

Reply:
I agree with the kaos of this page, I did spent a lot of time on it. 
I've added a new functionality to a dd a comment and I've made a little enhancement with the styles, I've also added a banner with an image. 

Respecta a clickar al nom del usuari i ensenyar la seva localització, amb les dades que torna la api ho tenia complicat. Cada post ve amb un email i un nom assignat, pero no corresponen als users que torna la api. 
La api torna 10 usuari (/users), que per aquest motiu en el input nomes deixo agregar numero del 1 al 10, y no son els mateixos usuaris que els dels comentaris. Ya vaig intentar buscar algun fetch amb el email pero no el tenen.

Reply:
Just saw the new functionality, great!

Yep, no problem, with the comments we haven't realized about the email thing, but with the post could be done. No need for you to do it now xD